.changeset/chatty-penguins-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-astro': minor
+---
+
+Defer async actions until after user input
```

Comment:
```suggestion
Improves the `create astro` CLI experience by asking all the questions upfront, then creating your new Astro project based on your responses.
```

I'd suggest something with a bit more context like this! These messages are user-facing, and we try to describe the change in user experience.